packages/examples/create-react-app/src/components/Realm.tsx

```diff
@@ -0,0 +1,37 @@
+import { useEffect } from "react";
```

Comment:
do you mean to include realms logic here? i think it would be best for the examples to use the simple example we have in the repo

Reply:
was just testing stuff, plan is to use the simple example with no realms logic